README.md

```diff
@@ -82,6 +82,7 @@ If you're looking for issues to help out with, check out [the issues labelled "g
 - [@wdio/selenium-standalone-service](https://github.com/webdriverio/webdriverio/tree/master/packages/wdio-selenium-standalone-service) - A WebdriverIO service that automatically sets up a selenium standalone server
 - [@wdio/browserstack-service](https://github.com/webdriverio/webdriverio/tree/master/packages/wdio-browserstack-service) - A WebdriverIO service that provides a better integration into Browserstack
 - [@wdio/crossbrowsertesting-service](https://github.com/webdriverio/webdriverio/tree/master/packages/wdio-crossbrowsertesting-service) - A WebdriverIO service that provides a better integration into CrossBrowserTesting
+- [wdio-lambdatest-service](https://github.com/LambdaTest/wdio-lambdatest-service) - A WebdriverIO service that provides a better integration into LambdaTest
```

Comment:
Given you host the service in your organisation we should remove this here.
```suggestion
```

Reply:
Removed